src/TensorFlowNET.Examples/Program.cs

```diff
@@ -62,37 +63,52 @@ static void Main(string[] args)
                 sw.Stop();
 
                 if (isSuccess)
+                {
                     success.Add($"Example: {name} in {sw.Elapsed.TotalSeconds}s");
+                    // only successfully run examples could be sorted as finished
+                    finished++;
+                }
                 else
                     errors.Add($"Example: {name} in {sw.Elapsed.TotalSeconds}s");
 
-                finished++;
+
                 keras.backend.clear_session();
             }
 
+            if (parsedArgs.ContainsKey("ex"))
+            {
+                printRunInfo(finished, examples.Length);
+                return;
+            }
+
             success.ForEach(x => Console.WriteLine($"{x} is OK!", Color.White));
             errors.ForEach(x => Console.WriteLine($"{x} is Failed!", Color.Red));
 
+            printRunInfo(finished, examples.Length);
+        }
+        private static void printRunInfo(int finished, int examplesLength)
+        {
             Console.WriteLine(Environment.OSVersion, Color.Yellow);
             Console.WriteLine($"64Bit Operating System: {Environment.Is64BitOperatingSystem}", Color.Yellow);
             Console.WriteLine($".NET CLR: {Environment.Version}", Color.Yellow);
             Console.WriteLine($"TensorFlow Binary v{tf.VERSION}");
             Console.WriteLine($"TensorFlow.NET v{Assembly.GetAssembly(typeof(TF_DataType)).GetName().Version}");
             Console.WriteLine($"TensorFlow.Keras v{Assembly.GetAssembly(typeof(KerasApi)).GetName().Version}");
-            Console.WriteLine($"{finished} of {examples.Length} example(s) are completed.");
+            Console.WriteLine($"{finished} of {examplesLength} example(s) are completed.");
```

Comment:
Is this a typo? should be `examples.Length`?

Reply:
It seems to be not a typo but the name of a parameter.